azerothcore/azerothcore-wotlk
# fix(DB/creature): Fix Deputy Feldon hostility against horde players

## Changes Proposed:
-  Update flag

## Issues Addressed:
- Closes : https://github.com/azerothcore/azerothcore-wotlk/issues/6775

## SOURCE:
https://wotlkdb.com/?npc=1070

## Tests Performed:
- Create horde character -> `.go c 6713` -> you should be attacked right away 
- Create alliance character -> `.go c 6713` -> `.character level 11` -> `.q add 244` and complete it and take next one (the guard itself is also a quest-giver for alliance players)


## How to Test the Changes:
- Create horde character -> `.go c 6713` -> you should be attacked right away 
- Create alliance character -> `.go c 6713` -> `.character level 11` -> `.q add 244` and complete it and take next one (the guard itself is also a quest-giver for alliance players)

## Known Issues and TODO List:
- none


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
